### PR TITLE
ImportC add C strings

### PR DESCRIPTION
**Pull request by WalterBright**
1. recognize `L`, `u` and `U` prefixed strings
2. fix Ddoc comment
3. disable `&entity;` recognition
4. don't allow newlines in strings

Doesn't seem to be a reasonable way to exercise this without semantic() support.

### CODE REVIEW

File: src/dmd/lexer.d [hunk 1]
```diff
@@ -449,9 +451,11 @@ class Lexer
                 }
                 else if (p[1] == '\"')  // C wide string literal
                 {
+                    const c = *p;
                     ++p;
                     escapeStringConstant(t);
-                    t.postfix = 't';
+                    t.postfix = c == 'L' ? (wchar_tsize == 2 ? 'w' : 'd') :
+                                c == 'u' ? 'w' : 'd';
```
Comment (thewilsonator) [inline src/dmd/lexer.d:458]: please split this nested ternary, it is very hard to follow.

**Review by thewilsonator — COMMENTED** [review]
Tests?

**Comment by WalterBright** [comment]
> Tests?

"Doesn't seem to be a reasonable way to exercise this without semantic() support."

File: src/dmd/lexer.d (hunk 1 above)
Comment (WalterBright) [reply src/dmd/lexer.d]: ok

**Comment by thewilsonator** [comment]
OK

**Review by RazvanN7 — APPROVED** [review]
